docs/release/release_0_5_0.md

```diff
@@ -0,0 +1,911 @@
+and application. Please read [NAP-6](nap-6-contributable-menus) for all the
+juicy details, including how to request more menus if the existing ones don't
+meet your needs. 📋 ([#7011](https://github.com/napari/napari/pull/7011))
+
+Another important development for plugins is that we have added fields for
+axis names and physical units in layers
+([#6979](https://github.com/napari/napari/pull/6979)). If you implement a
+reader plugin, you can now specify the names of the axes in the data that you
+are reading in, and the physical units of the scale and other transformations.
+Currently, napari is *not* using this information, but we will in upcoming
+versions, so plugins should start providing this information if they have it.
+
+There's plenty of new features, too, including a polygon drawing tool when
+painting labels ([#5806](https://github.com/napari/napari/pull/5806)),
+pinch-to-zoom ([#5859](https://github.com/napari/napari/pull/5859)), better
+ways to show/hide individual layers when exploring your data
+([#5574](https://github.com/napari/napari/pull/5574))
+([#5618](https://github.com/napari/napari/pull/5618)), and more:
+Over 20 new features in all and over 100 bug fixes and improvements!
+Please see below for the full list of changes since 0.4.19.
+
+## New Features
+
+- Add layer slicer base class for async slicing ([napari/napari#5170](https://github.com/napari/napari/pull/5170))
+- Feature: add layer contextual menu to hide not-selected layers ([napari/napari#5574](https://github.com/napari/napari/pull/5574))
+- Feature: alt-click visibility eye to toggle show just that layer (hide others) ([napari/napari#5618](https://github.com/napari/napari/pull/5618))
+- Feature: Add action to set current label to background (with keybinding) ([napari/napari#5672](https://github.com/napari/napari/pull/5672))
+- Use Dims margin in layer slicing code ([napari/napari#5697](https://github.com/napari/napari/pull/5697))
+- Changing brush size with mouse movement while holding specified modifiers ([napari/napari#5753](https://github.com/napari/napari/pull/5753))
+- Feature: polygon drawing in the Labels layer ([napari/napari#5806](https://github.com/napari/napari/pull/5806))
+- Replace old async loading with new approach ([napari/napari#5816](https://github.com/napari/napari/pull/5816))
+- Enable pinch-to-zoom with vispy 0.13+ ([napari/napari#5859](https://github.com/napari/napari/pull/5859))
+- Keep shading light for Surface fixed relative to camera ([napari/napari#5893](https://github.com/napari/napari/pull/5893))
+- Use Request/Response pattern for Vectors layer ([napari/napari#5918](https://github.com/napari/napari/pull/5918))
+- Lock dimensions / axes while rolling ([napari/napari#5986](https://github.com/napari/napari/pull/5986))
+- Added support for features in surface layers ([napari/napari#6515](https://github.com/napari/napari/pull/6515))
+- Add creating image from clipboard ([napari/napari#6532](https://github.com/napari/napari/pull/6532))
```

Comment:
To me, this one is 🔥 maybe worth highlight? if doing anything with remote data, like in S3 buckets, it's sick to be able to copy and open directly in napari -- assuming zarr/plugins anyways.